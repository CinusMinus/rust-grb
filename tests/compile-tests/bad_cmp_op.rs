use gurobi::*;
mod utils;

fn main() -> Result<()> {
  create_model!(_g, m, x, y, z);
  c!(x + y != 1 - z);
  c!(x + y < 1 - z);
  c!(x + y > 1 - z);
  Ok(())
}
