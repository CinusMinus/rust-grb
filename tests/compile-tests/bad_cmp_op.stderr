error: expected >=, <= or ==
 --> $DIR/bad_cmp_op.rs:7:12
  |
7 |   c!(x + y != 1 - z);
  |            ^^

error: expected >=, <= or ==
 --> $DIR/bad_cmp_op.rs:8:12
  |
8 |   c!(x + y < 1 - z);
  |            ^

error: expected >=, <= or ==
 --> $DIR/bad_cmp_op.rs:9:12
  |
9 |   c!(x + y > 1 - z);
  |            ^
